brave/brave-core
# (Wallet) Add market tab

## Resolves https://github.com/brave/brave-browser/issues/30301

Implements new market tab section showing the first 250 assets.
Implements shimmer effect while loading.

### Night Mode

https://github.com/brave/brave-core/assets/3308503/6cbc2b94-c24d-430d-9e14-5c27db3cdc18

### Day Mode

https://github.com/brave/brave-core/assets/3308503/1527ee4e-8e1c-4486-9eea-b167922dd500





## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

* Open Brave Wallet
* Navigate to Market section
* Observe the list is made up of 250 items
